### PR TITLE
feat: add a 'TAKE_FILE_OWNERSHIP` env var to disable chown

### PR DESCRIPTION
**Pull request by tomsquest**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,28 @@ docker run -d --name radicale \
     tomsquest/docker-radicale
 ```
   
-Note: On the Linux Kernel capabilities, `CHOWN`, `SETUID` and `SETGID` are for fixing the permission of the mounted volume. `KILL` is to allow Radicale to exit.
+Note on capabilities:
+- `CHOWN` is used to restore the permission of the `data` directory. Skip it if not chowning (see below)
+- `SETUID` and `SETGID` are used to changed the `UID` and `GID` of the user if asked. Skip them if not changing UID/GID (see below)
```
Comment (bernieke) [inline README.md:84]: SETUID and SETGID are required for su-exec to work, so are always needed.

**Review by bernieke — APPROVED** [review]
TAKE_FILE_OWNERSHIP works for me!

File: README.md (hunk 1 above)
Comment (tomsquest) [reply README.md]: Totally right ! :+1:
Fixed.

**Comment by tomsquest** [comment]
Released (being built) as 3.0.6.2

**Pull request merged**